django_tutorial/first_project/api_server/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+from api_server import views
+
+app_name = 'api_server'
+urlpatterns = [
+    url(r'^$', views.index, name='index'),
+    url(r'^v1/regions$', views.regions, name='regions'),
```

Comment:
Oh nevermind I think I get it. So in `first_project/urls.py` we have:

```
urlpatterns = [
    url(r'^api/', include('api_server.urls')),
    url(r'^admin/', admin.site.urls),
    url(r'^app/', include('web_app.urls')),
]
```

So basically this tells us that if we have a url that includes `api/` we also include the urls defined in `api_server/urls.py`. Similar for `web_app` right?